fuel/downloaders/base.py

```diff
@@ -55,6 +69,9 @@ def download(url, file_handle, chunk_size=1024):
         URL to download.
     file_handle : file
         Where to save the downloaded URL.
+    fd : open file handle, optional
```

Comment:
The reason it's confusing to me is that it's usually used for file handles,
but there's already a file handle argument. Maybe that's enough to tip off
someone to read the docstring, I dunno.

On Tue, May 19, 2015 at 6:56 PM, vdumoulin <notifications@github.com> wrote:

> In fuel/downloaders/base.py
> <https://github.com/mila-udem/fuel/pull/122#discussion_r30656783>:
>
> > @@ -55,6 +69,9 @@ def download(url, file_handle, chunk_size=1024):
> >          URL to download.
> >      file_handle : file
> >          Where to save the downloaded URL.
> > +    fd : open file handle, optional
>
> The reason for that is I re-used the same name as in progressbar2, as
> well as the same default value. I have nothing against giving it a better
> name and a saner default. Ideas?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/fuel/pull/122/files#r30656783>.
>